wandb/apis/public.py

```diff
@@ -1693,7 +1689,7 @@ def size(self):
     @retry.retriable(
         retry_timedelta=RETRY_TIMEDELTA,
         check_retry_fn=util.no_retry_auth,
-        retryable_exceptions=(RetryError, requests.RequestException),
+        retryable_exceptions=(requests.RequestException),
```

Comment:
This looks like you wanted this to be a tuple but it isnt.
did you want:
```
retryable_exceptions=(requests.RequestException,),
```

digging into where it is used, im not sure it matters, but all other users seem to use a tuple here.

Reply:
Good catch, it still worked because we just do `except self._retryable_exceptions as e` which will work with a single value or a tuple, but best to be consistent.  Just fixed.